kazoo/handlers/gevent.py

```diff
@@ -142,18 +143,20 @@ def stop(self):
             python2atexit.unregister(self.stop)
 
     def select(self, *args, **kwargs):
-        try:
-            return self._poll_select(*args, **kwargs)
-        except AttributeError:
-            return gevent.select.select(*args, **kwargs)
+        # if the highest fd we've seen is > 1023, use a poll-based "select".
+        if max(map(_to_fileno, chain(*args[:3]))) > 1023:
```

Comment:
Any reason not to use this as a default?

Reply:
Not really. But I want to make behaviour change as small as possible